workers/unity/Assets/Fps/Scripts/GameLogic/StateMachine/Session/PrepareDeploymentsListState.cs

```diff
@@ -44,30 +41,27 @@ private void BrowseDeployments()
 
         private void Connect()
         {
-            Owner.SetState(new ConnectState(deployment: null, Manager, Owner));
+            Owner.SetState(new SessionConnectState(deployment: null, Manager, Owner));
         }
 
         private bool PrepareDeploymentsList()
         {
-            if (Owner.Blackboard.LoginTokens == null)
+            if (Blackboard.LoginTokens == null)
             {
                 Owner.SetState(Owner.StartState);
             }
 
-            var hasAvailableDeployment = false;
             var deploymentData = new List<DeploymentData>();
-            foreach (var loginToken in Owner.Blackboard.LoginTokens)
+            foreach (var loginToken in Blackboard.LoginTokens)
             {
-                if (DeploymentData.TryToCreateFromTags(loginToken.DeploymentName, loginToken.Tags, out var data))
+                if (DeploymentData.TryFromTags(loginToken.DeploymentName, loginToken.Tags, out var data))
```

Comment:
Instead of expanding the arguments from `loginToken`: 

`TryFromTags` -> `TryFromLoginToken(LoginToken token, out DeploymentData data)`